tests/vmi_multus_test.go

```diff
@@ -1140,6 +1140,9 @@ var _ = Describe("[Serial]Macvtap", func() {
 
 func cidrToIP(cidr string) (string, error) {
 	ip, _, err := net.ParseCIDR(cidr)
+	if err != nil {
+		fmt.Errorf("Failed to parse IP from CIDR: %w", err)
```

Comment:
Could you please reason for this additional info? Why is the original error info not enough?

Reply:
@EdDev  I followed instructions from @phoracek to include this one. Please refer to resolved conversations.

Reply:
@EdDev squashed everything and removed this message.